hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsSqsProperties.kt

```diff
@@ -20,6 +20,7 @@ data class HmppsSqsProperties(
     val dlqAccessKeyId: String = "",
     val dlqSecretAccessKey: String = "",
     val dlqMaxReceiveCount: Int = 5,
+    val visibilityTimeout: Int = 30,
```

Comment:
default is 30 seconds in AWS, so duplicate that behaviour by default

Reply:
Can we document this in the README? Something to say that this is only relevant for testing with Localstack...